python/cudf/cudf/_libxx/parquet.pyx

```diff
@@ -35,6 +40,40 @@ from cudf._libxx.io.utils cimport (
 cimport cudf._libxx.cpp.types as cudf_types
 cimport cudf._libxx.cpp.io.types as cudf_io_types
 
+cdef class BufferArrayFromVector:
+    cdef unsigned length
```

Comment:
shouldn't this be `size_type`?
```suggestion
    cdef size_type length
```

Reply:
@brandon-b-miller  ?

Reply:
I'm actually not sure what the answer is here. I was following [this](https://cython.readthedocs.io/en/latest/src/userguide/buffer.html), where both `unsigned` and subsequently `Py_ssize_t` is used.

Reply:
I would use `Py_ssize_t` since this is a property used for Python Buffer protocol.